conans/client/command.py

```diff
@@ -2197,6 +2204,7 @@ def run(self, *args):
                     return False
 
                 self._warn_python_version()
+                self._warn_conan_version()
```

Comment:
I am a bit concerned that this might break some users parsing output from commands, that a new warning might disrupt them. Maybe add a warning only in ``conan install`` and ``conan create``?

Reply:
Makes sense, moved to install create and build methods